src/Symfony/Component/DependencyInjection/Dumper/PhpDumper.php

```diff
@@ -632,7 +633,7 @@ private function addService($id, Definition $definition)
         }
 
         if ($definition->isAutowired()) {
-            $doc = <<<EOF
+            $doc = <<<'EOF'
```

Comment:
@fabpot is it intended, even in php cs fixer, or is it a bug ? I don't see the value of quoting the heredoc